### DISCUSSION TITLE
Live data won't be available for pre-season testing in Bahrain

### DISCUSSION THREAD

**Discussion by br-g** [br-g/openf1#16] [Announcements]
Over the last weeks, I've been busy improving the API to make it more efficient and stable as we welcome more users.

The 3 days of pre-season testing in Bahrain will also be a test for OpenF1's new version; live data won't be available during this time but will return on March 7th for the Saudi Arabian GP.

Excited for the new season and to see what you are building with the API!

**Comment by DanieldeQuervain** [comment]
Thanks for the heads up! Looking forward to this season!

**Comment by Kloe** [comment]
Hello,
Are all data fetched from the API considered live data ?
For example, if I try to access the list of meetings and sessions for 2024, it is empty. I was thinking those would be static and not live data, is it normal and they would be filled after 7th of March ?
Thanks,
Chloe

**Reply by SimplyAG** [reply]
Im wondering the same thing, Need the race dates + times for 2024 season and I would believe these links would be where I could grab that info but they are empty:
[https://api.openf1.org/v1/sessions?year=2024](https://api.openf1.org/v1/sessions?year=2024)
https://api.openf1.org/v1/meetings?year=2024

Thanks!
PJs

**Reply by br-g** [reply]
Hello,
For now, only past events are provided by the API.
Future events (for the current season) will be added soon!
